### PR TITLE
Feat/see game after end

### PR DESCRIPTION
**Pull request by RomainGuarinoni**
As you can see below, the user can now see his game after the game is finish ! 

When the game is finish, the user can't change the latest target because the eventListeners has been disabled

seeResult

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -103,7 +106,12 @@ <h2>Dommage, tu as perdu</h2>
           <p>La combinaison était :</p>
           <div id="solution-combination"></div>
         </div>
-        <button id="loose-restart">Nouvelle partie</button>
+        <div class="button-container">
+          <button class="reduce-popUp" id="reduce-loose">
```
Comment (Guillaume-FAURE) [inline public/index.html:110]: lose instead of loose

**Review by Guillaume-FAURE — COMMENTED** [review]
great idea, perfect code, just spelling error on 'lose', sorry to be annoying

**Comment by Guillaume-FAURE** [comment]
don't wait for me to resolve conflict and merge branch

File: public/index.html (hunk 1 above)
Comment (RomainGuarinoni) [reply public/index.html]: Done

**Pull request merged**